tests/unit/__mocks__/axios.ts

```diff
@@ -0,0 +1,3 @@
+export default {
+  get: jest.fn(() => Promise.resolve({ data: {} })),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['jest' is not defined. (no-undef)](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4537582)